lib/rgeo/impl_helper/utils.rb

```diff
@@ -18,20 +18,14 @@ module Utils # :nodoc:
       # multiple times with different values and others pass the data
       # to a CAPI or FFI.
       def self.setup_coord_sys(srid, coord_sys, coord_sys_class)
-        unless coord_sys_class.is_a?(Class)
-          coord_sys_class = CoordSys::CONFIG.default_coord_sys_class
-        end
+        coord_sys_class = CoordSys::CONFIG.default_coord_sys_class unless coord_sys_class.is_a?(Class)
```

Comment:
This is just my 2c, but I tend to use 120 as the max. This will allow our editors of choice (RubyMine or VSCode) to have the nav menu open and the code still fit within the visible pane, but this will just depend on the size of the screen you're using. 120 is also the [default Rubocop value](https://docs.rubocop.org/rubocop/1.39/cops_layout.html#configurable-attributes-34).

But like I said, that is just how my company and I (on personal projects) do it. I know of other companies that limit it to 72 (not sure why tho, never thought to ask).

Reply:
72 comes from typography (https://ux.stackexchange.com/a/3623/105326), it is mostly good for comments or doc. Besides that, 120 is fine with me.

I guess the issue is for ppl who like split screens

Reply:
120 is also fine with me